### PR TITLE
Update certman to go1.13/operator-sdk 0.16

### PR DESCRIPTION
**Pull request by cblecker**
This updates certman to build on go1.13, and it's dependencies to be based off of operator-sdk 0.16.

As a side effect, it also moves it to use go modules, and removes the vendor directory.

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -79,56 +87,61 @@ func start() error {
...
 	if err != nil {
 		log.Error(err, "")
-		return err
+		os.Exit(1)
+	}
+
+	// Set default manager options
+	options := manager.Options{
+		Namespace:          namespace,
+		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
```
Comment (c-e-brumm) [inline cmd/manager/main.go:100]: Setting up the sdk default Metrics here conflicts with our custom metrics server that's defined below. Removing line 109 allows the operator to run as expected in test environments.

**Comment by cblecker** [comment]
coverage will fail until https://github.com/openshift/release/pull/7896 goes in

**Comment by cblecker** [comment]
/hold cancel
This one I think is ready, @c-e-brumm

**Pull request merged**